src/components/templates/Playa/VenuePreview.tsx

```diff
@@ -159,11 +160,14 @@ const VenuePreview: React.FC<VenuePreviewProps> = ({
       .then(function (array) {
         const futureEvents = array.docs
           .map((doc) => doc.data())
-          .filter((event) => event.start_utc_seconds > nowSeconds)
-          .sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
+          .filter((event) => isEventLiveOrFuture(event as VenueEvent));
 
         // TODO: is this type cast correct?
-        setEventsFuture(futureEvents as VenueEvent[]);
+        setEventsFuture(
+          sortEventsByStartUtcSeconds(
+            futureEvents as WithVenueId<WithId<VenueEvent>>[]
+          )
+        );
```

Comment:
[note] This is playa (legacy code to be deleted), so we don't mind that this doesn't fit our patterns/type safety.